webrtc.html

```diff
@@ -895,6 +899,20 @@
                     <var>connection</var>.</p>
                   </li>
                   <li>
+                    <p>If <var>description</var> is set as a local description,
+                    and it creates a new SCTP association, as defined in
+                    [[!SCTP-SDP]], set the value of <var>connection</var>'s
+                    [[<code><a>sctpTransport</a></code>]] internal slot
+                    to a newly created <code><a>RTCSctpTransport</a></code>.</p>
```

Comment:
I don't believe that a new SCTP association can be created until setRemoteDescription is called, because until then the remote port isn't known.  So you might say, "creates a new <code><a>RTCSctpTransport</a></code> object.".